radian/key_bindings.py

```diff
@@ -335,11 +337,86 @@ def create_key_bindings(editor=""):
     kb = KeyBindings()
     handle = kb.add
 
-    # emit completion
-    @handle('c-j', filter=insert_mode & default_focused & completion_is_selected)
-    @handle('enter', filter=insert_mode & default_focused & completion_is_selected)
+
+    def is_callable(text=""):
+        try:
+            return rcopy(rcall("class", reval(text))) == "function"
+        except:
+            return False
```

Comment:
I don't think it is a robust way to detect if a token is a function. We may need to poke into how the token is determined: https://github.com/randy3k/rchitect/blob/master/rchitect/completion.py#L18

Reply:
I agree that this part of the code could be improved, but so far the parentheses are added without fail. In other words, I have not found a situation in which this doesn't work. 
Since you are the author of both `rchitect` and `radian`, I defer to you regarding the best way to determine if a token is a function😄